diff -ru grub-2.12/util/grub-mkconfig.in grub-2.12.new/util/grub-mkconfig.in
--- grub-2.12/util/grub-mkconfig.in	2022-11-14 15:52:54.000000000 +0000
+++ grub-2.12.new/util/grub-mkconfig.in	2024-01-02 09:47:42.678935854 +0000
@@ -282,6 +282,10 @@
     *~) ;;
     # emacsen autosave files. FIXME: support other editors
     */\#*\#) ;;
+    # Incoming config files should be skipped as well:
+    *.new) ;;
+    # Outging config files should be skipped as well:
+    *.orig) ;;
     *)
       if grub_file_is_not_garbage "$i" && test -x "$i" ; then
         echo
