--- a/t/live-https.t
+++ b/t/live-https.t
@@ -51,12 +51,15 @@ SKIP: for my $pass ( 1 .. 2 ) {
         Accept            => '*/*'
     );
 
-    my ( $code, $mess, %h ) = $s->read_response_headers;
+    my ( $code, $mess, %headers ) = $s->read_response_headers;
+
+    my %h = map { lc($_) => $headers{$_} } keys %headers;
+
     print "# ----------------------------\n";
     print "# $code $mess\n";
     for ( sort keys %h ) {
         print "# $_: $h{$_}\n";
-        if (/^Connection$/i && $h{$_} =~ /^keep-alive$/) {
+        if (/^connection$/ && $h{$_} =~ /^keep-alive$/) {
             $connection_is_kept_alive = 1;
         }
     }
@@ -72,7 +75,7 @@ SKIP: for my $pass ( 1 .. 2 ) {
     $buf =~ s/\r//g;
 
     ok( $code == 302 || $code == 200, 'success' );
-    like( $h{'Content-Type'}, qr{text/html} );
+    like( $h{'content-type'}, qr{text/html} );
     like( $buf, qr{</html>}i );
 }
 
--- a/t/live.t
+++ b/t/live.t
@@ -6,13 +6,13 @@ BEGIN {
     eval {
         require IO::Socket::INET;
         my $s = IO::Socket::INET->new(
-            PeerHost => "www.neverssl.com:80",
+            PeerHost => "example.com:80",
             Timeout  => 5,
         );
         die "Can't connect: $@" unless $s;
     };
     if ($@) {
-        print "1..0 # SKIP Can't connect to www.neverssl.com\n";
+        print "1..0 # SKIP Can't connect to example.com\n";
         print $@;
         exit;
     }
@@ -26,7 +26,7 @@ use Test::More;
 use Net::HTTP;
 
 my $s = Net::HTTP->new(
-    Host            => "www.neverssl.com",
+    Host            => "example.com",
     KeepAlive       => 1,
     Timeout         => 15,
     PeerHTTPVersion => "1.1",
