diff -ru mate-control-center-1.28.0/shell/matecc.menu mate-control-center-1.28.0.new/shell/matecc.menu
--- mate-control-center-1.28.0/shell/matecc.menu	2024-02-22 18:36:56.000000000 +0100
+++ mate-control-center-1.28.0.new/shell/matecc.menu	2024-03-15 00:28:56.818413581 +0100
@@ -42,6 +42,10 @@
   </Include>
   <Exclude>
     <Filename>matecc.desktop</Filename>
+    <Filename>pavucontrol.desktop</Filename>
+    <Filename>fcitx-configtool.desktop</Filename>
+    <Filename>org.freedesktop.IBus.Setup.desktop</Filename>
+    <Filename>yad-settings.desktop</Filename>
   </Exclude>
 
   <!-- Avoid the shell having a launcher for itself -->
@@ -63,7 +67,20 @@
         <Category>System</Category>
         <Category>Printing</Category>
       </And>
+      <Filename>absm.deskop</Filename>
+      <Filename>gtklocalesetup.desktop</Filename>
+      <Filename>gtkusersetup.desktop</Filename>
+      <Filename>gtkclocksetup.desktop</Filename>
+      <Filename>gtkservicesetup.desktop</Filename>
+      <Filename>gslapt.desktop</Filename>
+      <Filename>dotnew.desktop</Filename>
+      <Filename>rescuebootstick.desktop</Filename>
+      <Filename>mate-system-monitor.desktop</Filename>
     </Include>
+    <Exclude>
+      <Filename>qcontrolcenter.desktop</Filename>
+      <Filename>re.sonny.Junction.desktop</Filename>
+    </Exclude>
   </Menu> <!-- End Administration -->
 
   <!-- Hardware category -->
@@ -76,6 +93,10 @@
         <Category>HardwareSettings</Category>
         <Not><Category>System</Category></Not>
       </And>
+      <Filename>pavucontrol.desktop</Filename>
+      <Filename>fcitx-configtool.desktop</Filename>
+      <Filename>org.freedesktop.IBus.Setup.desktop</Filename>
+      <Filename>hardinfo.desktop</Filename>
     </Include>
   </Menu> <!-- End Hardware -->
 
@@ -100,6 +121,9 @@
         <Not><Category>System</Category></Not>
       </And>
     </Include>
+    <Exclude>
+      <Filename>xscreensaver-settings.desktop</Filename>
+    </Exclude>
   </Menu> <!-- End Internet and Network -->
 
   <!-- Look and Feel category -->
@@ -120,6 +144,13 @@
         <Not><Category>Security</Category></Not>
       </And>
     </Include>
+    <Exclude>
+      <Filename>obconf.desktop</Filename>
+      <Filename>obconf-qt.desktop</Filename>
+      <Filename>pavucontrol-qt.desktop</Filename>
+      <Filename>kvantummanager.desktop</Filename>
+      <Filename>pcmanfm-qt-desktop-pref.desktop</Filename>
+    </Exclude>
   </Menu> <!-- End Look and Feel category -->
 
 
@@ -149,6 +180,9 @@
         <Not><Category>Utility</Category></Not>
       </And>
     </Include>
+    <Exclude>
+      <Filename>kvantummanager.desktop</Filename>
+    </Exclude>
   </Menu> <!-- End Personal -->
 
 </Menu> <!-- End CC -->
