diff -ru slapt-get-git3e25960/src/main.c slapt-get-git3e25960.new/src/main.c
--- slapt-get-git3e25960/src/main.c	2020-01-25 22:11:13.000000000 +0100
+++ slapt-get-git3e25960.new/src/main.c	2021-01-08 23:03:00.111781668 +0100
@@ -37,7 +37,6 @@
         {"upgrade", 0, 0, SLAPT_UPGRADE_OPT},
         {"install", 0, 0, SLAPT_INSTALL_OPT},
         {"i", 0, 0, SLAPT_INSTALL_OPT},
-        {"remove", 0, 0, SLAPT_REMOVE_OPT},
         {"show", 0, 0, SLAPT_SHOW_OPT},
         {"search", 0, 0, SLAPT_SEARCH_OPT},
         {"list", 0, 0, SLAPT_LIST_OPT},
@@ -115,9 +114,6 @@
         case SLAPT_INSTALL_OPT: /* install */
             do_action = SLAPT_ACTION_INSTALL;
             break;
-        case SLAPT_REMOVE_OPT: /* remove */
-            do_action = SLAPT_ACTION_REMOVE;
-            break;
         case SLAPT_SHOW_OPT: /* show */
             do_action = SLAPT_ACTION_SHOW;
             initial_config->simulate = true; /* allow read access */
@@ -427,7 +423,6 @@
     printf("  --dist-upgrade %s\n", gettext("upgrade to newer release"));
     printf("  -i, --install  %s\n", gettext("[pkg name(s)] - install specified pkg(s)"));
     printf("  --install-set  %s\n", gettext("[disk set(s)] - install specified disk set(s)"));
-    printf("  --remove       %s\n", gettext("[pkg name(s)] - remove specified pkg(s)"));
     printf("  --show         %s\n", gettext("[pkg name(s)] - show pkg(s) description"));
     printf("  --filelist     %s\n", gettext("[pkg name(s)] - show pkg(s) installed files"));
     printf("  --search       %s\n", gettext("[expression] - search available pkgs"));
