diff -ru slapt-get-git3e25960/src/package.h slapt-get-git3e25960.new/src/package.h
--- slapt-get-git3e25960/src/package.h	2020-01-25 22:11:13.000000000 +0100
+++ slapt-get-git3e25960.new/src/package.h	2020-08-28 23:31:24.549249407 +0200
@@ -45,7 +45,7 @@
 #define SLAPT_PATCHDIR "patches/"
 #define SLAPT_REMOVE_CMD "/sbin/removepkg "
 #define SLAPT_INSTALL_CMD "/sbin/installpkg "
-#define SLAPT_UPGRADE_CMD "/sbin/upgradepkg --reinstall "
+#define SLAPT_UPGRADE_CMD "/sbin/wrapupgradepkg --reinstall "
 #define SLAPT_CHECKSUM_FILE "CHECKSUMS.md5"
 #define SLAPT_CHECKSUM_FILE_GZ "CHECKSUMS.md5.gz"
 #define SLAPT_HEAD_FILE_EXT ".head"
