diff -ru sourcery-0.9.6/src/sourcery sourcery-0.9.6.new/src/sourcery
--- sourcery-0.9.6/src/sourcery	2016-08-04 17:39:01.000000000 +0200
+++ sourcery-0.9.6.new/src/sourcery	2022-04-15 22:17:43.157236675 +0200
@@ -27,7 +27,7 @@
 # configuration options
 config_file = '/etc/sourcery.conf'
 slaptsrcrc = '/etc/slapt-get/slapt-srcrc'
-slackbuilds_data = '/usr/src/slapt-src/slackbuilds_data'
+slackbuilds_data = '/var/lib/slapt-src/slackbuilds_data'
 logdir = '/var/log/sourcery'
 shareddir = '/usr/share/sourcery'
 
