diff -aur mate-tweak-18.04.16/data/mate-tweak.ui mate-tweak-18.04.16.new/data/mate-tweak.ui
--- mate-tweak-18.04.16/data/mate-tweak.ui	2018-03-29 01:12:44.000000000 +0200
+++ mate-tweak-18.04.16.new/data/mate-tweak.ui	2018-05-08 21:59:35.758688744 +0200
@@ -40,7 +40,7 @@
             <child>
               <object class="GtkIconView" id="side_view">
                 <property name="visible">True</property>
-                <property name="can_focus">False</property>
+                <property name="can_focus">True</property>
                 <property name="margin">0</property>
                 <property name="border_width">6</property>
                 <property name="columns">1</property>
