Description: Fix the nim root in nim-gdb
Author: Gabriel Huber <mail@gabrielhuber.at>
Forwarded: not-needed

--- a/bin/nim-gdb
+++ b/bin/nim-gdb
@@ -10,7 +10,7 @@ which readlink > /dev/null || (echo "readlink not in PATH."; exit 1)
 if [[ $(uname -s) == Darwin || $(uname -s) == *BSD ]]; then
   NIM_SYSROOT=$(dirname $(dirname $(readlink -f $(which nim))))
 else
-  NIM_SYSROOT=$(dirname $(dirname $(readlink -e $(which nim))))
+  NIM_SYSROOT=$(dirname $(dirname $(readlink -e $(which nim))))/lib/nim
 fi
 
 # Find out where the pretty printer Python module is
