diff -Naur libunique-1.1.6-old/configure libunique-1.1.6/configure
--- libunique-1.1.6-old/configure	2009-11-12 15:57:56.000000000 +0200
+++ libunique-1.1.6/configure	2012-07-18 20:32:47.065940845 +0300
@@ -15398,7 +15398,7 @@
 if test "x$enable_maintainer_flags" = "xyes" &&  test "x$GCC" = "xyes"; then
 
 
-  list="-Werror -Wall -Wshadow -Wcast-align
+  list="-Wall -Wshadow -Wcast-align
                             -Wno-uninitialized -Wempty-body -Wformat-security
                             -Winit-self"
   flags_supported=""
diff -Naur libunique-1.1.6-old/configure.ac libunique-1.1.6/configure.ac
--- libunique-1.1.6-old/configure.ac	2009-11-12 15:57:39.000000000 +0200
+++ libunique-1.1.6/configure.ac	2012-07-18 20:31:42.410365534 +0300
@@ -178,7 +178,7 @@
 AS_IF([test "x$enable_maintainer_flags" = "xyes" &&  test "x$GCC" = "xyes"],
       [
         AS_COMPILER_FLAGS([MAINTAINER_CFLAGS],
-                          ["-Werror -Wall -Wshadow -Wcast-align
+                          ["-Wall -Wshadow -Wcast-align
                             -Wno-uninitialized -Wempty-body -Wformat-security
                             -Winit-self"])
       ]
