diff -Naru RHVoice-git20181213/src/sd_module/list_voices.cpp RHVoice-git20181213.new/src/sd_module/list_voices.cpp
--- RHVoice-git20181213/src/sd_module/list_voices.cpp	2018-12-09 13:39:36.000000000 +0100
+++ RHVoice-git20181213.new/src/sd_module/list_voices.cpp	2018-12-14 10:32:52.692588723 +0100
@@ -19,15 +19,6 @@
 #include "list_voices.hpp"
 #include "io.hpp"
 
-namespace
-{
-#if ((SPD_MAJOR==0) && (SPD_MINOR<9))
-  const char sep=' ';
-#else
-  const char sep='\t';
-  #endif
-}
-
 namespace RHVoice
 {
   namespace sd
@@ -55,7 +46,7 @@
                   language_code=it->primary()->get_language()->get_alpha2_code();
                   if(language_code.empty())
                     language_code="none";
-                  r("200-",voice_name,sep,language_code,sep,dialect);
+                  r("200-",voice_name,'\t',language_code,'\t',dialect);
                 }
               r("200 OK VOICE LIST SENT");
             }
