diff -ru i38/README.md i38.new/README.md
--- i38/README.md	2023-04-10 03:48:11.000000000 +0200
+++ i38.new/README.md	2023-04-10 10:09:56.380458224 +0200
@@ -39,8 +39,6 @@
 - With no arguments, create the i3 configuration.
 - -h: This help screen.
 - -u: Copy over the latest version of scripts.
-- -x: Generate ~/.xinitrc and ~/.xprofile.
-- -X: Generate ~/.xprofile only.
 
 
 ## Ratpoison Mode
diff -ru i38/i38.sh i38.new/i38.sh
--- i38/i38.sh	2023-04-10 03:48:11.000000000 +0200
+++ i38.new/i38.sh	2023-04-10 12:12:31.437982068 +0200
@@ -134,8 +134,8 @@
     [h]="This help screen."
     [s]="Create sway configuration instead of i3."
     [u]="Copy over the latest version of scripts."
-    [x]="Generate ~/.xinitrc and ~/.xprofile."
-    [X]="Generate ~/.xprofile only."
+#    [x]="Generate ~/.xinitrc and ~/.xprofile."
+#    [X]="Generate ~/.xprofile only."
 )
 
 # Convert the keys of the associative array to a format usable by getopts
@@ -144,13 +144,13 @@
 while getopts "${args}" i ; do
     case "$i" in
         h) help;;
-        s)
-            i3msg="swaymsg"
-            i3Path="${XDG_CONFIG_HOME:-$HOME/.config}/sway"
-        ;;
+#        s)
+#            i3msg="swaymsg"
+#            i3Path="${XDG_CONFIG_HOME:-$HOME/.config}/sway"
+#        ;;
         u) update_scripts;;
-        x) write_xinitrc ;&
-        X) write_xprofile ;;
+#        x) write_xinitrc ;&
+#        X) write_xprofile ;;
     esac
 done
 
@@ -174,7 +174,7 @@
 volumeJump=$(rangebox "How much should pressing the volume keys change the volume?" 1 15 5)
 # Web browser
 unset programList
-for i in brave chromium epiphany firefox google-chrome-stable midori seamonkey ; do
+for i in brave chromium epiphany firefox google-chrome-stable midori seamonkey vivaldi; do
     if command -v ${i/#-/} &> /dev/null ; then
         if [ -n "$programList" ]; then
             programList="$programList $i"
@@ -191,7 +191,7 @@
 webBrowser="$(command -v $webBrowser)"
 # Text editor
 unset programList
-for i in emacs geany gedit kate kwrite l3afpad leafpad libreoffice mousepad pluma ; do
+for i in emacs geany gedit gvim kate kwrite l3afpad leafpad libreoffice mousepad pluma; do
 if hash ${i/#-/} &> /dev/null ; then
 if [ -n "$programList" ]; then
 programList="$programList $i"
@@ -226,14 +226,14 @@
 fileBrowser="$(command -v $fileBrowser)"
 # Auto start with dex
 dex=1
-if command -v dex &> /dev/null ; then
-    export dex=$(yesno "Would you like to autostart applications with dex?")
-fi
+#if command -v dex &> /dev/null ; then
+#    export dex=$(yesno "Would you like to autostart applications with dex?")
+#fi
 if [[ $dex -eq 0 ]]; then
     dex -t "${XDG_CONFIG_HOME:-${HOME}/.config}/autostart" -c $(command -v orca)
 fi
 brlapi=1
-brlapi=$(yesno "Do you want to use a braille display with Orca?")
+#brlapi=$(yesno "Do you want to use a braille display with Orca?")
 sounds=1
 sounds=$(yesno "Do you want window event sounds?")
 
@@ -465,7 +465,7 @@
 else
     echo '# Startup applications'
     echo 'exec --no-startup-id clipster -d'
-    echo 'exec orca'
+    echo 'exec --no-startup-id dex -a -s ~/.config/autostart/'
 fi)
 
 # If you want to add personal customizations to i3, add them in ${i3Path}/customizations
