diff -ru depfinder-1.4.4/depfinder depfinder-1.4.4.new/depfinder
--- depfinder-1.4.4/depfinder	2016-09-28 12:56:18.000000000 +0200
+++ depfinder-1.4.4.new/depfinder	2022-07-13 00:34:17.635470793 +0200
@@ -221,7 +221,7 @@
 
 # Create temp dir and extract package
 mkdir -p /tmp/depfinder.$$
-ls /var/log/packages > /tmp/depfinder.$$/pkglist.log
+ls /var/lib/pkgtools/packages > /tmp/depfinder.$$/pkglist.log
 
 # Extract only if argument is not a directory
 if [ `echo $1 | grep "\.t[gb]z$"` ]; then
@@ -407,7 +407,7 @@
 	sed ':a;s/\([^ \|]\+\)-\(\([^ \|-]\+\)-\([^ \|-]\+\)-\([^ \|-]\+\)\|?\)/\1 >= \2/g;ta' \
 	> DEPSNAMES
 	for i in `echo $ADD | sed "s/,/ /g"`; do
-		DEPVER=`ls /var/log/packages/$i-* | sed "s|/var/log/packages/$i-||" | grep "^[^\-]*\-[^\-]*\-[^\-]*$"`
+		DEPVER=`ls /var/lib/pkgtools/packages/$i-* | sed "s|/var/lib/pkgtools/packages/$i-||" | grep "^[^\-]*\-[^\-]*\-[^\-]*$"`
 		sed -i "s/^$i$/$i >= $DEPVER/" DEPSNAMES
 	done
 else
