diff -ru slapt-src-0.3.6/src/main.c slapt-src-0.3.6.new/src/main.c
--- slapt-src-0.3.6/src/main.c	2021-03-18 02:23:46.000000000 +0100
+++ slapt-src-0.3.6.new/src/main.c	2021-12-13 23:32:21.650224977 +0100
@@ -37,6 +37,7 @@
 
 static int show_summary(slapt_vector_t *, slapt_vector_t *, int, bool);
 static void clean(slapt_src_config *config);
+char command[54];
 
 void version(void)
 {
@@ -354,6 +355,10 @@
     case UPDATE_OPT:
         if (!slapt_src_update_slackbuild_cache(config))
             exit(EXIT_FAILURE);
+        if (access("/usr/libexec/slapt-src/remove-unwanted-slackbuilds_data",X_OK)==0) {
+            strcpy( command, "/usr/libexec/slapt-src/remove-unwanted-slackbuilds_data" );
+            system(command);
+        }
         break;
 
     case FETCH_OPT:
