diff -Naur slapt-src-0.3.7-old/src/source.c slapt-src-0.3.7/src/source.c
--- slapt-src-0.3.7-old/src/source.c	2022-02-05 21:35:37.000000000 +0200
+++ slapt-src-0.3.7/src/source.c	2022-03-20 20:33:42.919634560 +0200
@@ -777,12 +777,12 @@
         exit(EXIT_FAILURE);
     }
 
-    int command_len = 44;
+    int command_len = 41;
     char *filename = _get_pkg_filename(sb->version, config->pkgtag);
     if (filename != NULL) {
         command_len += strlen(filename);
         char *command = slapt_malloc(sizeof *command * command_len);
-        const int snprintf_r = snprintf(command, command_len, "/sbin/upgradepkg --reinstall --install-new %s", filename);
+        const int snprintf_r = snprintf(command, command_len, "/sbin/spkg -u --reinstall --install-new %s", filename);
         if (snprintf_r + 1 != command_len) {
             printf(gettext("Failed to construct command string\n"));
             exit(EXIT_FAILURE);
