--- LOG_059	2024-09-23 12:16:24.487795558 +0200
+++ LOG	2024-09-23 12:17:41.529802573 +0200
@@ -5,60 +5,64 @@
 cc -c -O2 -g -Wall -std=gnu99 -D_FILE_OFFSET_BITS=64 -Wformat -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2   src/install/util.c -o src/install/util.o
 cc -c -O2 -g -Wall -std=gnu99 -D_FILE_OFFSET_BITS=64 -Wformat -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2   src/skipcpio/skipcpio.c -o src/skipcpio/skipcpio.o
 cc -c -O2 -g -Wall -std=gnu99 -D_FILE_OFFSET_BITS=64 -Wformat -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2   src/util/util.c -o src/util/util.o
-asciidoc -a "version=" -d manpage -b docbook -o "man/lsinitrd.1.xml" man/lsinitrd.1.asc
-asciidoc -a "version=" -d manpage -b docbook -o "man/dracut.conf.5.xml" man/dracut.conf.5.asc
-asciidoc -a "version=" -d manpage -b docbook -o "man/dracut.cmdline.7.xml" man/dracut.cmdline.7.asc
-asciidoc -a "version=" -d manpage -b docbook -o "man/dracut.bootup.7.xml" man/dracut.bootup.7.asc
-asciidoc -a "version=" -d manpage -b docbook -o "man/dracut.modules.7.xml" man/dracut.modules.7.asc
-asciidoc -a "version=" -d manpage -b docbook -o "man/dracut.8.xml" man/dracut.8.asc
-asciidoc -a "version=" -d manpage -b docbook -o "man/dracut-catimages.8.xml" man/dracut-catimages.8.asc
-asciidoc -a "version=" -d manpage -b docbook -o "modules.d/98dracut-systemd/dracut-cmdline.service.8.xml" modules.d/98dracut-systemd/dracut-cmdline.service.8.asc
-asciidoc -a "version=" -d manpage -b docbook -o "modules.d/98dracut-systemd/dracut-initqueue.service.8.xml" modules.d/98dracut-systemd/dracut-initqueue.service.8.asc
-asciidoc -a "version=" -d manpage -b docbook -o "modules.d/98dracut-systemd/dracut-mount.service.8.xml" modules.d/98dracut-systemd/dracut-mount.service.8.asc
-asciidoc -a "version=" -d manpage -b docbook -o "modules.d/98dracut-systemd/dracut-shutdown.service.8.xml" modules.d/98dracut-systemd/dracut-shutdown.service.8.asc
-asciidoc -a "version=" -d manpage -b docbook -o "modules.d/98dracut-systemd/dracut-pre-mount.service.8.xml" modules.d/98dracut-systemd/dracut-pre-mount.service.8.asc
-asciidoc -a "version=" -d manpage -b docbook -o "modules.d/98dracut-systemd/dracut-pre-pivot.service.8.xml" modules.d/98dracut-systemd/dracut-pre-pivot.service.8.asc
-asciidoc -a "version=" -d manpage -b docbook -o "modules.d/98dracut-systemd/dracut-pre-trigger.service.8.xml" modules.d/98dracut-systemd/dracut-pre-trigger.service.8.asc
-asciidoc -a "version=" -d manpage -b docbook -o "modules.d/98dracut-systemd/dracut-pre-udev.service.8.xml" modules.d/98dracut-systemd/dracut-pre-udev.service.8.asc
+asciidoc -a "version=103" -d manpage -b docbook -o "man/lsinitrd.1.xml" man/lsinitrd.1.asc
+asciidoc -a "version=103" -d manpage -b docbook -o "man/dracut.conf.5.xml" man/dracut.conf.5.asc
+asciidoc -a "version=103" -d manpage -b docbook -o "man/dracut.cmdline.7.xml" man/dracut.cmdline.7.asc
+asciidoc -a "version=103" -d manpage -b docbook -o "man/dracut.bootup.7.xml" man/dracut.bootup.7.asc
+asciidoc -a "version=103" -d manpage -b docbook -o "man/dracut.modules.7.xml" man/dracut.modules.7.asc
+asciidoc -a "version=103" -d manpage -b docbook -o "man/dracut.8.xml" man/dracut.8.asc
+asciidoc -a "version=103" -d manpage -b docbook -o "man/dracut-catimages.8.xml" man/dracut-catimages.8.asc
+asciidoc -a "version=103" -d manpage -b docbook -o "modules.d/98dracut-systemd/dracut-cmdline.service.8.xml" modules.d/98dracut-systemd/dracut-cmdline.service.8.asc
+asciidoc -a "version=103" -d manpage -b docbook -o "modules.d/98dracut-systemd/dracut-initqueue.service.8.xml" modules.d/98dracut-systemd/dracut-initqueue.service.8.asc
+asciidoc -a "version=103" -d manpage -b docbook -o "modules.d/98dracut-systemd/dracut-mount.service.8.xml" modules.d/98dracut-systemd/dracut-mount.service.8.asc
+asciidoc -a "version=103" -d manpage -b docbook -o "modules.d/98dracut-systemd/dracut-shutdown.service.8.xml" modules.d/98dracut-systemd/dracut-shutdown.service.8.asc
+asciidoc -a "version=103" -d manpage -b docbook -o "modules.d/98dracut-systemd/dracut-pre-mount.service.8.xml" modules.d/98dracut-systemd/dracut-pre-mount.service.8.asc
+asciidoc -a "version=103" -d manpage -b docbook -o "modules.d/98dracut-systemd/dracut-pre-pivot.service.8.xml" modules.d/98dracut-systemd/dracut-pre-pivot.service.8.asc
+asciidoc -a "version=103" -d manpage -b docbook -o "modules.d/98dracut-systemd/dracut-pre-trigger.service.8.xml" modules.d/98dracut-systemd/dracut-pre-trigger.service.8.asc
+asciidoc -a "version=103" -d manpage -b docbook -o "modules.d/98dracut-systemd/dracut-pre-udev.service.8.xml" modules.d/98dracut-systemd/dracut-pre-udev.service.8.asc
 cc   src/skipcpio/skipcpio.o   -o src/skipcpio/skipcpio
 cc   src/util/util.o   -o src/util/util
 xsltproc -o "man/lsinitrd.1" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl man/lsinitrd.1.xml
 xsltproc -o "man/dracut.conf.5" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl man/dracut.conf.5.xml
 xsltproc -o "man/dracut.bootup.7" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl man/dracut.bootup.7.xml
 xsltproc -o "man/dracut.modules.7" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl man/dracut.modules.7.xml
-xsltproc -o "man/dracut-catimages.8" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl man/dracut-catimages.8.xml
 xsltproc -o "modules.d/98dracut-systemd/dracut-cmdline.service.8" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl modules.d/98dracut-systemd/dracut-cmdline.service.8.xml
+xsltproc -o "man/dracut-catimages.8" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl man/dracut-catimages.8.xml
+xsltproc -o "modules.d/98dracut-systemd/dracut-initqueue.service.8" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl modules.d/98dracut-systemd/dracut-initqueue.service.8.xml
+asciidoc: WARNING: dracut.cmdline.7.asc: line 1095: missing style: [blockdef-listing]: listing
+asciidoc: WARNING: dracut.cmdline.7.asc: line 1111: missing style: [blockdef-listing]: listing
 Note: Writing lsinitrd.1
 Note: Writing dracut.conf.5
-xsltproc -o "modules.d/98dracut-systemd/dracut-initqueue.service.8" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl modules.d/98dracut-systemd/dracut-initqueue.service.8.xml
 xsltproc -o "modules.d/98dracut-systemd/dracut-mount.service.8" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl modules.d/98dracut-systemd/dracut-mount.service.8.xml
 xsltproc -o "modules.d/98dracut-systemd/dracut-shutdown.service.8" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl modules.d/98dracut-systemd/dracut-shutdown.service.8.xml
 Note: Writing dracut.bootup.7
-Note: Writing dracut-catimages.8
-xsltproc -o "modules.d/98dracut-systemd/dracut-pre-mount.service.8" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl modules.d/98dracut-systemd/dracut-pre-mount.service.8.xml
 Note: Writing dracut.modules.7
 Note: Writing dracut-cmdline.service.8
+Note: Writing dracut-catimages.8
+xsltproc -o "modules.d/98dracut-systemd/dracut-pre-mount.service.8" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl modules.d/98dracut-systemd/dracut-pre-mount.service.8.xml
 xsltproc -o "modules.d/98dracut-systemd/dracut-pre-pivot.service.8" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl modules.d/98dracut-systemd/dracut-pre-pivot.service.8.xml
-cc  -o src/install/dracut-install src/install/dracut-install.o src/install/hashmap.o src/install/log.o src/install/strv.o src/install/util.o  -lc -lkmod
 xsltproc -o "modules.d/98dracut-systemd/dracut-pre-trigger.service.8" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl modules.d/98dracut-systemd/dracut-pre-trigger.service.8.xml
 xsltproc -o "modules.d/98dracut-systemd/dracut-pre-udev.service.8" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl modules.d/98dracut-systemd/dracut-pre-udev.service.8.xml
-cp -a src/util/util dracut-util
+cc  -o src/install/dracut-install src/install/dracut-install.o src/install/hashmap.o src/install/log.o src/install/strv.o src/install/util.o  -lc -lkmod
 Note: Writing dracut-initqueue.service.8
+cp -a src/util/util dracut-util
+xsltproc -o "man/dracut.8" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl man/dracut.8.xml
 xsltproc -o "man/dracut.cmdline.7" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl man/dracut.cmdline.7.xml
+ln -fs src/install/dracut-install dracut-install
 Note: Writing dracut-mount.service.8
 Note: Writing dracut-shutdown.service.8
-xsltproc -o "man/dracut.8" -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl man/dracut.8.xml
-ln -fs src/install/dracut-install dracut-install
 Note: Writing dracut-pre-mount.service.8
+Note: Writing dracut-pre-pivot.service.8
 Note: Writing dracut-pre-trigger.service.8
 Note: Writing dracut-pre-udev.service.8
-Note: Writing dracut-pre-pivot.service.8
 Note: Writing dracut.8
 Note: Writing dracut.cmdline.7
-asciidoc -a "mainversion=" \
-	-a "version=" \
+asciidoc -a "mainversion=103" \
+	-a "version=103" \
 	-a numbered \
 	-d book -b docbook -o dracut.xml man/dracut.asc
+asciidoc: WARNING: dracut.cmdline.7.asc: line 1095: missing style: [blockdef-listing]: listing
+asciidoc: WARNING: dracut.cmdline.7.asc: line 1111: missing style: [blockdef-listing]: listing
 xsltproc -o dracut.html --xinclude -nonet \
 	--stringparam custom.css.source docs/dracut.css \
 	--stringparam generate.css.header 1 \
@@ -159,7 +163,7 @@
 directory ownerships to root.root ([y]es, [n]o)? 
 n
 
-Creating Slackware package:  /repo/x86_64/slint-15.0/source/dracut/dracut-059-x86_64-1slint.txz
+Creating Slackware package:  /repo/x86_64/slint-15.0/source/dracut/dracut-103-x86_64-1slint.txz
 
 ./
 etc/
@@ -174,18 +178,20 @@
 usr/bin/dracut-catimages
 usr/bin/lsinitrd
 usr/doc/
-usr/doc/dracut-059/
-usr/doc/dracut-059/AUTHORS
-usr/doc/dracut-059/COPYING
-usr/doc/dracut-059/README.html
-usr/doc/dracut-059/dracut.SlackBuild
-usr/doc/dracut-059/dracut.conf.d/
-usr/doc/dracut-059/dracut.conf.d/fedora.conf.example
-usr/doc/dracut-059/dracut.conf.d/fips.conf.example
-usr/doc/dracut-059/dracut.conf.d/gentoo-systemd.conf.example
-usr/doc/dracut-059/dracut.conf.d/gentoo.conf.example
-usr/doc/dracut-059/dracut.conf.d/suse.conf.example
-usr/doc/dracut-059/dracut.html
+usr/doc/dracut-103/
+usr/doc/dracut-103/AUTHORS
+usr/doc/dracut-103/COPYING
+usr/doc/dracut-103/README.html
+usr/doc/dracut-103/dracut.SlackBuild
+usr/doc/dracut-103/dracut.conf.d/
+usr/doc/dracut-103/dracut.conf.d/50-fips.conf.example
+usr/doc/dracut-103/dracut.conf.d/50-generic.conf.example
+usr/doc/dracut-103/dracut.conf.d/50-hostonly.conf.example
+usr/doc/dracut-103/dracut.conf.d/50-ima.conf.example
+usr/doc/dracut-103/dracut.conf.d/50-rescue.conf.example
+usr/doc/dracut-103/dracut.conf.d/fedora.conf.example
+usr/doc/dracut-103/dracut.conf.d/suse.conf.example
+usr/doc/dracut-103/dracut.html
 usr/lib/
 usr/lib/kernel/
 usr/lib/kernel/install.d/
@@ -228,8 +234,12 @@
 usr/lib64/dracut/modules.d/01systemd-ac-power/module-setup.sh
 usr/lib64/dracut/modules.d/01systemd-ask-password/
 usr/lib64/dracut/modules.d/01systemd-ask-password/module-setup.sh
+usr/lib64/dracut/modules.d/01systemd-bsod/
+usr/lib64/dracut/modules.d/01systemd-bsod/module-setup.sh
 usr/lib64/dracut/modules.d/01systemd-coredump/
 usr/lib64/dracut/modules.d/01systemd-coredump/module-setup.sh
+usr/lib64/dracut/modules.d/01systemd-creds/
+usr/lib64/dracut/modules.d/01systemd-creds/module-setup.sh
 usr/lib64/dracut/modules.d/01systemd-hostnamed/
 usr/lib64/dracut/modules.d/01systemd-hostnamed/99-systemd-networkd-dracut.conf
 usr/lib64/dracut/modules.d/01systemd-hostnamed/module-setup.sh
@@ -247,7 +257,11 @@
 usr/lib64/dracut/modules.d/01systemd-modules-load/
 usr/lib64/dracut/modules.d/01systemd-modules-load/module-setup.sh
 usr/lib64/dracut/modules.d/01systemd-networkd/
+usr/lib64/dracut/modules.d/01systemd-networkd/99-default.network
+usr/lib64/dracut/modules.d/01systemd-networkd/99-wait-online-dracut.conf
 usr/lib64/dracut/modules.d/01systemd-networkd/module-setup.sh
+usr/lib64/dracut/modules.d/01systemd-networkd/networkd-config.sh
+usr/lib64/dracut/modules.d/01systemd-networkd/networkd-run.sh
 usr/lib64/dracut/modules.d/01systemd-pcrphase/
 usr/lib64/dracut/modules.d/01systemd-pcrphase/module-setup.sh
 usr/lib64/dracut/modules.d/01systemd-portabled/
@@ -259,8 +273,6 @@
 usr/lib64/dracut/modules.d/01systemd-resolved/
 usr/lib64/dracut/modules.d/01systemd-resolved/module-setup.sh
 usr/lib64/dracut/modules.d/01systemd-resolved/resolved-tmpfile-dracut.conf
-usr/lib64/dracut/modules.d/01systemd-rfkill/
-usr/lib64/dracut/modules.d/01systemd-rfkill/module-setup.sh
 usr/lib64/dracut/modules.d/01systemd-sysctl/
 usr/lib64/dracut/modules.d/01systemd-sysctl/module-setup.sh
 usr/lib64/dracut/modules.d/01systemd-sysext/
@@ -322,7 +334,6 @@
 usr/lib64/dracut/modules.d/35connman/cm-run.sh
 usr/lib64/dracut/modules.d/35connman/cm-wait-online-initrd.service
 usr/lib64/dracut/modules.d/35connman/module-setup.sh
-usr/lib64/dracut/modules.d/35connman/netroot.sh
 usr/lib64/dracut/modules.d/35network-legacy/
 usr/lib64/dracut/modules.d/35network-legacy/dhclient-script.sh
 usr/lib64/dracut/modules.d/35network-legacy/dhclient.conf
@@ -346,19 +357,17 @@
 usr/lib64/dracut/modules.d/35network-manager/nm-lib.sh
 usr/lib64/dracut/modules.d/35network-manager/nm-run.sh
 usr/lib64/dracut/modules.d/35network-manager/nm-wait-online-initrd.service
-usr/lib64/dracut/modules.d/35network-wicked/
-usr/lib64/dracut/modules.d/35network-wicked/module-setup.sh
-usr/lib64/dracut/modules.d/35network-wicked/wicked-config.sh
-usr/lib64/dracut/modules.d/35network-wicked/wicked-run.sh
 usr/lib64/dracut/modules.d/40network/
-usr/lib64/dracut/modules.d/40network/dhcp-root.sh
-usr/lib64/dracut/modules.d/40network/ifname-genrules.sh
 usr/lib64/dracut/modules.d/40network/module-setup.sh
-usr/lib64/dracut/modules.d/40network/net-lib.sh
-usr/lib64/dracut/modules.d/40network/netroot.sh
 usr/lib64/dracut/modules.d/45ifcfg/
 usr/lib64/dracut/modules.d/45ifcfg/module-setup.sh
 usr/lib64/dracut/modules.d/45ifcfg/write-ifcfg.sh
+usr/lib64/dracut/modules.d/45net-lib/
+usr/lib64/dracut/modules.d/45net-lib/dhcp-root.sh
+usr/lib64/dracut/modules.d/45net-lib/ifname-genrules.sh
+usr/lib64/dracut/modules.d/45net-lib/module-setup.sh
+usr/lib64/dracut/modules.d/45net-lib/net-lib.sh
+usr/lib64/dracut/modules.d/45net-lib/netroot.sh
 usr/lib64/dracut/modules.d/45url-lib/
 usr/lib64/dracut/modules.d/45url-lib/module-setup.sh
 usr/lib64/dracut/modules.d/45url-lib/url-lib.sh
@@ -391,6 +400,7 @@
 usr/lib64/dracut/modules.d/80test-makeroot/module-setup.sh
 usr/lib64/dracut/modules.d/80test-root/
 usr/lib64/dracut/modules.d/80test-root/module-setup.sh
+usr/lib64/dracut/modules.d/80test-root/test-init.sh
 usr/lib64/dracut/modules.d/80test/hard-off.sh
 usr/lib64/dracut/modules.d/80test/module-setup.sh
 usr/lib64/dracut/modules.d/81cio_ignore/
@@ -479,11 +489,17 @@
 usr/lib64/dracut/modules.d/90multipath/multipathd-stop.sh
 usr/lib64/dracut/modules.d/90multipath/multipathd.service
 usr/lib64/dracut/modules.d/90multipath/multipathd.sh
+usr/lib64/dracut/modules.d/90numlock/
+usr/lib64/dracut/modules.d/90numlock/module-setup.sh
+usr/lib64/dracut/modules.d/90numlock/numlock.sh
 usr/lib64/dracut/modules.d/90nvdimm/
 usr/lib64/dracut/modules.d/90nvdimm/module-setup.sh
 usr/lib64/dracut/modules.d/90overlayfs/
 usr/lib64/dracut/modules.d/90overlayfs/module-setup.sh
 usr/lib64/dracut/modules.d/90overlayfs/mount-overlayfs.sh
+usr/lib64/dracut/modules.d/90overlayfs/prepare-overlayfs.sh
+usr/lib64/dracut/modules.d/90pcmcia/
+usr/lib64/dracut/modules.d/90pcmcia/module-setup.sh
 usr/lib64/dracut/modules.d/90ppcmac/
 usr/lib64/dracut/modules.d/90ppcmac/load-thermal.sh
 usr/lib64/dracut/modules.d/90ppcmac/module-setup.sh
@@ -491,6 +507,8 @@
 usr/lib64/dracut/modules.d/90qemu-net/
 usr/lib64/dracut/modules.d/90qemu-net/module-setup.sh
 usr/lib64/dracut/modules.d/90qemu/module-setup.sh
+usr/lib64/dracut/modules.d/90systemd-cryptsetup/
+usr/lib64/dracut/modules.d/90systemd-cryptsetup/module-setup.sh
 usr/lib64/dracut/modules.d/91crypt-gpg/
 usr/lib64/dracut/modules.d/91crypt-gpg/README
 usr/lib64/dracut/modules.d/91crypt-gpg/crypt-gpg-lib.sh
@@ -519,13 +537,9 @@
 usr/lib64/dracut/modules.d/95cifs/parse-cifsroot.sh
 usr/lib64/dracut/modules.d/95dasd/
 usr/lib64/dracut/modules.d/95dasd/module-setup.sh
-usr/lib64/dracut/modules.d/95dasd/parse-dasd.sh
 usr/lib64/dracut/modules.d/95dasd_mod/
 usr/lib64/dracut/modules.d/95dasd_mod/module-setup.sh
 usr/lib64/dracut/modules.d/95dasd_mod/parse-dasd-mod.sh
-usr/lib64/dracut/modules.d/95dasd_rules/
-usr/lib64/dracut/modules.d/95dasd_rules/module-setup.sh
-usr/lib64/dracut/modules.d/95dasd_rules/parse-dasd.sh
 usr/lib64/dracut/modules.d/95dcssblk/
 usr/lib64/dracut/modules.d/95dcssblk/module-setup.sh
 usr/lib64/dracut/modules.d/95dcssblk/parse-dcssblk.sh
@@ -545,6 +559,8 @@
 usr/lib64/dracut/modules.d/95fstab-sys/
 usr/lib64/dracut/modules.d/95fstab-sys/module-setup.sh
 usr/lib64/dracut/modules.d/95fstab-sys/mount-sys.sh
+usr/lib64/dracut/modules.d/95hwdb/
+usr/lib64/dracut/modules.d/95hwdb/module-setup.sh
 usr/lib64/dracut/modules.d/95iscsi/
 usr/lib64/dracut/modules.d/95iscsi/cleanup-iscsi.sh
 usr/lib64/dracut/modules.d/95iscsi/iscsiroot.sh
@@ -571,10 +587,9 @@
 usr/lib64/dracut/modules.d/95nvmf/
 usr/lib64/dracut/modules.d/95nvmf/95-nvmf-initqueue.rules
 usr/lib64/dracut/modules.d/95nvmf/module-setup.sh
+usr/lib64/dracut/modules.d/95nvmf/nbftroot.sh
 usr/lib64/dracut/modules.d/95nvmf/nvmf-autoconnect.sh
 usr/lib64/dracut/modules.d/95nvmf/parse-nvmf-boot-connections.sh
-usr/lib64/dracut/modules.d/95qeth_rules/
-usr/lib64/dracut/modules.d/95qeth_rules/module-setup.sh
 usr/lib64/dracut/modules.d/95resume/
 usr/lib64/dracut/modules.d/95resume/module-setup.sh
 usr/lib64/dracut/modules.d/95resume/parse-resume.sh
@@ -604,9 +619,6 @@
 usr/lib64/dracut/modules.d/95zfcp/
 usr/lib64/dracut/modules.d/95zfcp/module-setup.sh
 usr/lib64/dracut/modules.d/95zfcp/parse-zfcp.sh
-usr/lib64/dracut/modules.d/95zfcp_rules/
-usr/lib64/dracut/modules.d/95zfcp_rules/module-setup.sh
-usr/lib64/dracut/modules.d/95zfcp_rules/parse-zfcp.sh
 usr/lib64/dracut/modules.d/95znet/
 usr/lib64/dracut/modules.d/95znet/module-setup.sh
 usr/lib64/dracut/modules.d/95znet/parse-ccw.sh
@@ -660,6 +672,7 @@
 usr/lib64/dracut/modules.d/98dracut-systemd/dracut-tmpfiles.conf
 usr/lib64/dracut/modules.d/98dracut-systemd/emergency.service
 usr/lib64/dracut/modules.d/98dracut-systemd/module-setup.sh
+usr/lib64/dracut/modules.d/98dracut-systemd/parse-root.sh
 usr/lib64/dracut/modules.d/98dracut-systemd/rootfs-generator.sh
 usr/lib64/dracut/modules.d/98ecryptfs/
 usr/lib64/dracut/modules.d/98ecryptfs/README
@@ -746,4 +759,4 @@
 usr/share/pkgconfig/
 usr/share/pkgconfig/dracut.pc
 
-Slackware package /repo/x86_64/slint-15.0/source/dracut/dracut-059-x86_64-1slint.txz created.
+Slackware package /repo/x86_64/slint-15.0/source/dracut/dracut-103-x86_64-1slint.txz created.
