diff -ru i38/i38.sh i38.new/i38.sh
--- i38/i38.sh	2024-06-30 03:52:19.000000000 +0200
+++ i38.new/i38.sh	2024-07-02 10:57:20.667854340 +0200
@@ -312,7 +312,7 @@
 emailClient="$(command -v $emailClient)"
 # Web browser
 unset programList
-for i in brave chromium epiphany firefox google-chrome-stable google-chrome-unstable microsoft-edge-stable microsoft-edge-beta microsoft-edge-dev midori seamonkey ; do
+for i in brave chromium epiphany firefox google-chrome-stable google-chrome-unstable microsoft-edge-stable microsoft-edge-beta microsoft-edge-dev midori seamonkey vivaldi; do
     if command -v ${i/#-/} &> /dev/null ; then
         if [ -n "$programList" ]; then
             programList="$programList $i"
@@ -329,7 +329,7 @@
 webBrowser="$(command -v $webBrowser)"
 # Text editor
 unset programList
-for i in emacs geany gedit kate kwrite l3afpad leafpad libreoffice mousepad pluma ; do
+for i in emacs geany gedit gvim kate kwrite l3afpad leafpad libreoffice mousepad pluma ; do
 if hash ${i/#-/} &> /dev/null ; then
 if [ -n "$programList" ]; then
 programList="$programList $i"
@@ -372,15 +372,15 @@
 if command -v dex &> /dev/null ; then
     export dex=$(yesno "Would you like to autostart applications with dex?")
 fi
-if [[ $dex -eq 0 ]]; then
-    dex -t "${XDG_CONFIG_HOME:-${HOME}/.config}/autostart" -c $(command -v orca)
-fi
+#if [[ $dex -eq 0 ]]; then
+#    dex -t "${XDG_CONFIG_HOME:-${HOME}/.config}/autostart" -c $(command -v orca)
+#fi
 if command -v acpi &> /dev/null ; then
     batteryAlert=1
     batteryAlert=$(yesno "Do you want low battery notifications?")
 fi
 brlapi=1
-brlapi=$(yesno "Do you want to use a braille display with Orca?")
+#brlapi=$(yesno "Do you want to use a braille display with Orca?")
 sounds=1
 sounds=$(yesno "Do you want window event sounds?")
 # Play Login Sound
@@ -661,7 +661,7 @@
 else
     echo '# Startup applications'
     echo 'exec --no-startup-id clipster -d'
-    echo 'exec orca'
+    echo 'exec --no-startup-id dex -a -s ~/.config/autostart/'
     echo "exec_always --no-startup-id ${i3Path}/scripts/desktop.sh"
 fi)
 
