diff -ru abcde-2.9.3/abcde.conf abcde-2.9.3.new/abcde.conf
--- abcde-2.9.3/abcde.conf	2019-02-05 11:19:03.000000000 +0100
+++ abcde-2.9.3.new/abcde.conf	2023-07-28 09:54:55.106505567 +0200
@@ -10,12 +10,12 @@
 # "cdtext". Default is "musicbrainz", but all can be specified in a
 # comma delimited list to be tried sequentially. All the results will
 # be displayed ready for user choice.
-#CDDBMETHOD=musicbrainz
+CDDBMETHOD=CDDB
 
 # If you wish to use a different CDDB server, edit this line.
 # If you just wanted to use a proxy server, just set your http_proxy
 # environment variable - wget will use it correctly.
-#CDDBURL="http://freedb.freedb.org/~cddb/cddb.cgi"
+CDDBURL="http://gnudb.gnudb.org/~cddb/cddb.cgi"
 
 # The CDDB protocol level.
 # Right now 5 is latin1 output and 6 is UTF8 encoding.
