diff -ru mate-panel-1.28.2.orig/applets/clock/clock.c mate-panel-1.28.2/applets/clock/clock.c
--- mate-panel-1.28.2.orig/applets/clock/clock.c	2024-06-18 21:43:26.000000000 +0200
+++ mate-panel-1.28.2/applets/clock/clock.c	2024-10-15 22:07:05.205090520 +0200
@@ -1782,7 +1782,7 @@
                 gtk_widget_set_sensitive (cd->time_settings_button, can_set);
 
         if (cd->set_time_button) {
-                gtk_widget_set_sensitive (cd->set_time_button, can_set != 0);
+                gtk_widget_set_sensitive (cd->set_time_button, TRUE);
                 gtk_button_set_label (GTK_BUTTON (cd->set_time_button),
                                       can_set == 1 ?
                                         _("Set System Time...") :
