diff -ru mate-panel-1.28.0/mate-panel/panel-icon-names.h mate-panel-1.28.0.new/mate-panel/panel-icon-names.h
--- mate-panel-1.28.0/mate-panel/panel-icon-names.h	2024-02-20 02:30:10.000000000 +0100
+++ mate-panel-1.28.0.new/mate-panel/panel-icon-names.h	2024-03-31 18:08:40.009804231 +0200
@@ -21,7 +21,7 @@
 #define PANEL_ICON_LAUNCHER		"mate-panel-launcher"
 #define PANEL_ICON_LOCKSCREEN		"system-lock-screen"
 #define PANEL_ICON_LOGOUT		"system-log-out"
-#define PANEL_ICON_MAIN_MENU		"start-here"
+#define PANEL_ICON_MAIN_MENU		"start-here-slint"
 #define PANEL_ICON_NETWORK		"network-workgroup"
 #define PANEL_ICON_NETWORK_SERVER	"network-server"
 #define PANEL_ICON_PANEL		"mate-panel"
