diff --git a/t/g.t b/t/g.t
index 6390326..3de9e22 100644
--- a/t/g.t
+++ b/t/g.t
@@ -96,7 +96,7 @@ SKIP: {
 	$key_file->set_comment('locales', 'mystring', 'comment');
 	like ($key_file->get_comment('locales', 'mystring'), qr/^comment$/);
 	$key_file->set_comment('locales', undef, "another comment\n");
-	is ($key_file->get_comment('locales', undef), "#another comment\n#");
+	like ($key_file->get_comment('locales', undef), qr/^#?another comment\n#?$/);
 	$key_file->set_comment(undef, undef, 'one comment more');
 	like ($key_file->get_comment(undef, undef), qr/^one comment more$/);
 	$key_file->set_boolean($start_group, 'boolkey', FALSE);
