diff -ru compiz-core-v0.8.18/metadata/switcher.xml.in compiz-core-v0.8.18.new/metadata/switcher.xml.in
--- compiz-core-v0.8.18/metadata/switcher.xml.in	2020-04-04 20:24:41.000000000 +0200
+++ compiz-core-v0.8.18.new/metadata/switcher.xml.in	2024-02-21 21:03:43.568962699 +0100
@@ -32,7 +32,6 @@
 		<option name="next_all_key" type="key">
 		    <short>Next window (All windows)</short>
 		    <long>Popup switcher if not visible and select next window out of all windows</long>
-		    <default>&lt;Primary&gt;&lt;Alt&gt;Tab</default>
 		</option>
 		<option name="prev_all_button" type="button">
 		    <short>Prev window (All windows)</short>
@@ -41,7 +40,6 @@
 		<option name="prev_all_key" type="key">
 		    <short>Prev window (All windows)</short>
 		    <long>Popup switcher if not visible and select previous window out of all windows</long>
-		    <default>&lt;Shift&gt;&lt;Primary&gt;&lt;Alt&gt;Tab</default>
 		</option>
 		<option name="next_no_popup_button" type="button">
 		    <short>Next window (No popup)</short>
@@ -66,6 +64,7 @@
 		<option name="next_panel_key" type="key">
 		    <short>Next Panel</short>
 		    <long>Select next panel type window.</long>
+		    <default>&lt;Control&gt;&lt;Primary&gt;&lt;Alt&gt;Tab</default>
 		</option>
 		<option name="prev_panel_button" type="button">
 		    <short>Prev Panel</short>
@@ -74,6 +73,7 @@
 		<option name="prev_panel_key" type="key">
 		    <short>Prev Panel</short>
 		    <long>Select previous panel type window.</long>
+		    <default>&lt;Shift&gt;&lt;Control&gt;&lt;Primary&gt;&lt;Alt&gt;Tab</default>
 		</option>
 	    </group>
 	</display>
