diff -aru salixtools-1.11.3/keyboardsetup/keyboardsetup salixtools-1.11.3.new/keyboardsetup/keyboardsetup
--- salixtools-1.11.3/keyboardsetup/keyboardsetup	2018-01-01 19:04:04.000000000 +0100
+++ salixtools-1.11.3.new/keyboardsetup/keyboardsetup	2020-01-16 15:32:11.853576094 +0100
@@ -418,7 +418,7 @@
 		[[ $DISPLAY ]] && [ -x /usr/bin/setxkbmap ] && \
 		/usr/bin/setxkbmap -layout "$xkblayout" -variant "$xkbvariant" -option "$xkboptions" 1>&2 2>/dev/null
 	fi
-	sed -i "s/\(^[ \t]*\/usr\/bin\/loadkeys -u\).*$/\1 $keymap/" /etc/rc.d/rc.keymap
+	sed -i "s/\(^[[:space:]]*\/usr\/bin\/loadkeys\).*$/\1 $keymap/" /etc/rc.d/rc.keymap
 
 	# set keymap in /etc/X11/xorg.conf.d/50-keymap.conf
 	mkdir -p /etc/X11/xorg.conf.d
