diff -Naru pulseaudio-12.2/src/daemon/default.pa.in pulseaudio-12.2.new/src/daemon/default.pa.in
--- pulseaudio-12.2/src/daemon/default.pa.in	2018-07-13 21:06:14.000000000 +0200
+++ pulseaudio-12.2.new/src/daemon/default.pa.in	2018-12-14 10:14:27.056984088 +0100
@@ -52,6 +52,11 @@
 #load-module module-pipe-sink
 ])dnl
 
+### In Slint, we want to share audio resources between speech apps that
+### rely on alsa and other apps that rely on pulseaudio.  
+load-module module-alsa-sink device=dmix
+load-module module-alsa-source device=dsnoop
+
 ### Automatically load driver modules depending on the hardware available
 ifelse(@HAVE_UDEV@, 1, [dnl
 .ifexists module-udev-detect@PA_SOEXT@
